joschahenningsen/TUM-Live
# confirm before deleting lecture

### Motivation and Context


Closes #1227 
Deleting a lecture and it's associated recording should require user confirmation.

### Description

Added a confirm dialogue to deleting a lecture.

### Steps for Testing

Prerequisites:
- 1 Lecturer
- 1 Lecture

1. Log in as an admin
2. Navigate to a Course
3. Delete a Lecture, notice it first asks for confirmation

### Screenshots

![Screenshot_20231122_144559](https://github.com/TUM-Dev/gocast/assets/61728152/72759365-ee00-4653-9991-9c309284c5b3)
